mlflow/utils/_capture_modules.py

```diff
@@ -84,6 +85,11 @@ def main():
 
     cap_cm = _CaptureImportedModules()
 
+    if flavor == "spark" and is_in_databricks_runtime():
+        from dbruntime.spark_connection import initialize_spark_connection
+
+        initialize_spark_connection()
```

Comment:
if the case not in databricks runtime ?

Reply:
@WeichenXu123 

If not, a new spark session is created in the following code:
https://github.com/mlflow/mlflow/blob/c4b8e849a3c718575e106f3499a12bea9b0b821e/mlflow/spark.py#L687-L700

Reply:
Shall we add try/catch here and add fallback handling ?
e.g. the case  User upgrade/downgrade the builtin mlflow of databrick runtime and found the `dbruntime.spark_connection.initialize_spark_connection` API does not exist.

Reply:
Agree with @WeichenXu123 , I think a try/catch is a good idea here.